test/reduce-e2e/testdata/joined-map-vertex-pipeline.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: joined-map-vertex
+spec:
+  limits:
+    readBatchSize: 50
+  vertices:
+    - name: in-0
+      scale:
+        min: 1
+      containerTemplate:
+        env:
+          - name: NUMAFLOW_DEBUG
```

Comment:
Why do we need to enable debug in the e2e test?

Reply:
meant to remove all of that - was using it for my own debugging - will do